### PR TITLE
Skip the get_raw_tables test if python doesn't  have memoryview

### PR DESCRIPTION
**Pull request by kdmurray91**
This addresses @luizirber's comment in #868 that python2.6 chokes on this test.

### CODE REVIEW

**Comment by kdmurray91** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by kdmurray91** [comment]
Ready for review

**Comment by mr-c** [comment]
Thanks @kdmurray91. I, and others, don't automatically get notified unless we are already subscribed to an issue so an @ mention will get our attention :-)

**Comment by ctb** [comment]
ping @luizirber can you check this on 2.6?

Also, @mr-c, what do we want to do about 2.7.2 vs 2.7.6? The error in #868 is seen in 2.7.2.  I think we should put in the same SkipTest if we get a type error on the memoryview call, but alternative suggestions welcome.

**Comment by mr-c** [comment]
The skiptest is fine; it is an experimental & unsupported API either way :-)

**Comment by ctb** [comment]
I've set up a PR #871 against this branch that makes the tests pass on Python 2.6.2.

**Comment by kdmurray91** [comment]
Rebased on master, should be mergeable.

File: ChangeLog [hunk 1]
```diff
@@ -1,8 +1,12 @@
+2015-03-15  Kevin Murray  <spam@kdmurray.id.au>  &  Titus Brown  <titus@idyll.org>
```
Comment (kdmurray91) [inline ChangeLog]: @mr-c  @ctb  is this the right way to credit 2 people in a CL entry?

Comment (ctb) [inline ChangeLog]: On Sat, Mar 14, 2015 at 03:57:28PM -0700, Kevin Murray wrote:
> > @@ -1,8 +1,12 @@
> > +2015-03-15  Kevin Murray  <spam@kdmurray.id.au>  &  Titus Brown  <titus@idyll.org>
> 
> @mr-c  @ctb  is this the right way to credit 2 people in a CL entry?

Oh, no need to credit... seriously... :)

**Comment by kdmurray91** [comment]
Can confirm this works w/ 2.7.9 and 2.7.6, which is all I have access to.

**Comment by mr-c** [comment]
LGTM

**Pull request merged**